src/videojs.ads.js

```diff
@@ -668,8 +668,8 @@ var
           enter: function() {
             this.snapshot = getPlayerSnapshot(player);
             if (player.ads.nopostroll_) {
-              player.ads.state = 'content-resuming';
               window.setTimeout(function() {
+                player.ads.state = 'content-resuming';
```

Comment:
Should we note why this is happening in a setTimeout() somewhere? This is definitely in the non-obvious category for me.